docs/decisions/0002-extracts-plugins-infrastructure-from-edx-platform.rst

```diff
@@ -0,0 +1,24 @@
+1. Extract plugins Infrastructure from edx-platform
+===================================================
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+We have experimented with adding plugins to edx-platform and are now ready to move this functionality to other IDAs. However, since the plugins infrastructure is currently located in edx-platform and thus not easily usable by IDAs.
+
+
+
+Decision
+--------
+
+Move plugin infrastructure to edx-django-utils.
```

Comment:
Maybe add a note that we decided this didn't warrant its own repo at the time, but that could be considered if it was deemed important for other reasons in the future.